src/components/hero.jsx

```diff
@@ -0,0 +1,143 @@
+import { cva } from "class-variance-authority";
+import PropTypes from "prop-types";
+
+import Button from "./button";
+import Heading from "./heading";
+
+const heroVariants = cva(
+  "mx-auto flex w-full max-w-[1360px] flex-col gap-8 sm:flex-row",
+  {
+    variants: {
+      layout: {
+        leftAligned: "items-start justify-between",
+        centered: "flex-col items-center justify-center text-center",
+      },
+    },
+    defaultVariants: {
+      layout: "leftAligned",
+    },
+  },
+);
+
+const Hero = ({
+  layout,
+  preHeading,
+  heading,
+  headingSize = "Large",
+  headingElement,
+  textColor = "Dark",
+  text,
+  button1Label,
+  button1Link,
+  button1Style = "Solid",
```

Comment:
Switching naming conventions between style and variant causes a little mental context switching. Thoughts about continuing to use variant, such as `button1Variant`?

Additionally, these props are pretty flat, which is fine but would there be any improvements to architecture if objects were used to group like items?

Example, button1 would pass through an object `{ label, link, variant } `, same as button2